upgrade-2.x.sh

```diff
@@ -815,16 +815,18 @@ fi
 
 progress 25 "Preparing OS update"
 
-# Check board support
+# Check board support (explicit support only needed for device types with pre-hostapp versions available)
 case $SLUG in
-    artik53*|artik710|asus-tinker-board*|bananapi-m1-plus|beaglebone*|cl-som-imx8|fincm3|hummingboard|imx*-var-dart|jetson-tx1|jetson-tx2|odroid-c1|odroid-xu4|orangepi-plus2|orbitty-tx2|raspberry*|revpi-core-3|skx2|ts4900)
+    artik710|beaglebone*|imx6-var-dart|kitra710|raspberry*|ts4900)
         binary_type=arm
         ;;
     intel-edison|intel-nuc|iot2000|up-board|qemux86*)
         binary_type=x86
         ;;
     *)
-        if { version_gt "$target_version" "$minimum_hostapp_target_version" || [ "$target_version" == "$minimum_hostapp_target_version" ]; } &&
+        if version_gt "${VERSION_ID}" "${minimum_hostapp_target_version}" || [ "${VERSION_ID}" == "${minimum_hostapp_target_version}" ] ; then
+            log "Update implicitly supported with hostapp-based updates for ${SLUG} device type."
+        elif { version_gt "$target_version" "$minimum_hostapp_target_version" || [ "$target_version" == "$minimum_hostapp_target_version" ]; } &&
```

Comment:
small nit: can we style the variables consistently here?
```suggestion
        elif { version_gt "${target_version}" "${minimum_hostapp_target_version}" || [ "${target_version}" == "${minimum_hostapp_target_version}" ]; } &&
```

Reply:
@xginn8 fair, should have done it already... Will do that. 
Also, these scripts would need some bit of housekeeping (of not real changes to functionality, just mostly cleaning up - hopefully some day soon)

Reply:
@xginn8 pushed changes as per your uggestion